lib/src/checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+import DxcCheckbox from "./Checkbox";
+import { BackgroundColorProvider } from "../BackgroundColorContext";
+import Title from "../../.storybook/components/Title";
+import ExampleContainer from "../../.storybook/components/ExampleContainer";
+import DarkContainer from "../../.storybook/components/DarkSection";
+import { userEvent, within } from "@storybook/testing-library";
+
+export default {
+  title: "Checkbox",
+  component: DxcCheckbox,
+};
+
+const Checkbox = () => (
+  <>
+    <>
+      <ExampleContainer>
+        <Title title="Default" theme="light" level={4} />
+        <DxcCheckbox label="Checkbox" />
+      </ExampleContainer>
+      <ExampleContainer pseudoState="pseudo-focus">
```

Comment:
If we are simulating getting the focus through tabulation with userEvent, I think this pseudoState could be deleted.

Reply:
Yes! I forgot to delete it. ✌🏼